AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/HTMLTemplate.java

```diff
@@ -52,6 +56,10 @@ public HTMLTemplate(Activity activity, HTMLOptions htmlOptions) {
   @Override
   public boolean dispatchTouchEvent(@NonNull MotionEvent ev) {
     if (!htmlOptions.isFullScreen()) {
+      if (treatAsBanner(htmlOptions)) {
+        return super.dispatchTouchEvent(ev);
```

Comment:
if it's not a banner, do the touches go through without this?

Reply:
not so sure what you mean? if this condition is not satisfied the touch events will still be dispatched